### PR TITLE
feat(launcher): Set default Firefox prefs (#5149)

### PR DESCRIPTION
**Pull request by mjzffr**
Add recommended automation preferences to profile
setup when launching Firefox. This profile can be overridden
by using the `userDataDir` launch option, or individual prefs
can be overwritten with the `extraPrefsFirefox` option.

The preferences have been reviewed by peers at Mozilla
over at https://bugzilla.mozilla.org/show_bug.cgi?id=1596888

### CODE REVIEW

File: lib/Launcher.js
Comment (mathiasbynens) [inline lib/Launcher.js]:
```suggestion
      'network.sntp.pools': server,
```

**Review by mathiasbynens — APPROVED** [review]
The change looks good. I fully defer to your expertise w.r.t. Firefox preferences.

File: lib/Launcher.js
Comment (mathiasbynens) [reply lib/Launcher.js]: I would have committed this minor change myself but I don't have [push permissions to your PR branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

Comment (mjzffr) [reply lib/Launcher.js]: Huh, that's odd. I had "Allow edits" checked.

Comment (mathiasbynens) [reply lib/Launcher.js]: Glitch in the GitHub matrix

**Pull request merged**
